src/openzaak/components/documenten/api/filters.py

```diff
@@ -30,13 +33,119 @@
 logger = logging.getLogger(__name__)
 
 
+class ZaakFilter(FkOrUrlFieldFilter):
+    def filter(self, qs, value):
+        if not value:
+            return qs
+
+        if not check_path(value, "zaken"):
```

Comment:
I don't quite understand why you need this/ We don't have any special conditions for other `FkOrUrlFieldFilter` filters